src/mod_auth_cas.c

```diff
@@ -1844,8 +1856,10 @@ char *getResponseFromServer (request_rec *r, cas_cfg *c, char *ticket)
 	curl_easy_setopt(curl, CURLOPT_URL, apr_uri_unparse(r->pool, &validateURL, 0));
 
 	if(curl_easy_perform(curl) != CURLE_OK) {
-		if(c->CASDebug)
+		if(c->CASDebug) {
+			ap_log_rerror(APLOG_MARK, APLOG_DEBUG, 0, r, "MOD_AUTH_CAS: query: %s", validateURL.query);
```

Comment:
I'd probably move the previous log addition to line 1855 after validateURL.query is set so we get the whole validateURL printed out.

Reply:
This was moved.  Good catch.

Reply:
The log at line 1858 is redundant since it's logged at 1852 now.